app/controller/sdl/RCModulesController.js

```diff
@@ -0,0 +1,946 @@
+      return "Module ID: " + moduleGuid;
+    }.property('SDL.RCModulesController.currentClimateModel.UUID'),
+
+    /**
+     * @function getAudioCurrentID
+     * @description callback to get current module id name for Audio Views
+     */
+    getAudioCurrentID: function() {
+      var moduleGuid = SDL.RCModulesController.currentAudioModel.UUID ?
+      SDL.RCModulesController.currentAudioModel.UUID : 'DRIVER';
+      return "Module ID: " + moduleGuid;
+    }.property('SDL.RCModulesController.currentAudioModel.UUID'),
+
+    /**
+     * @function getRadioCurrentID
+     * @description callback to get current module id name for Radio View
+     */
+    getRadioCurrentID: function() {
+      var moduleGuid = SDL.RCModulesController.currentRadioModel.UUID ?
+      SDL.RCModulesController.currentRadioModel.UUID : 'DRIVER';
+      return "Module ID: " + moduleGuid;
+    }.property('SDL.RCModulesController.currentRadioModel.UUID'),
+
+    /**
+     * @function changeCurrentModule
+     * @description Function invoked when user changes a seat zone
+     * @param {String} user_friendly_key
+     */
+    changeCurrentModule: function(user_friendly_key) {
+      Object.keys(this.seatKeyLabelMapping).forEach(
+        key => {
+          var value = this.seatKeyLabelMapping[key];
+          if (value == user_friendly_key) {
+            this.updateCurrentModels(key);
+          }
+      });
+    },
+
+    /**
+     * @function updateCurrentModels
+     * @description Function to update current models according to a newly
+     * selected module
+     * @param {String} module_key
+     */
+    updateCurrentModels: function(module_key) {
+        this.set('currentClimateModel', this.getCoveringModuleModel('CLIMATE', module_key));
+        this.set('currentAudioModel', this.getCoveringModuleModel('AUDIO', module_key));
+        this.set('currentSeatModel', this.getCoveringModuleModel('SEAT', module_key));
+        this.set('currentRadioModel', this.getCoveringModuleModel('RADIO', module_key));
+        this.set('currentHMISettingsModel', this.getCoveringModuleModel('HMI_SETTINGS', module_key));
+        this.set('currentLightModel', this.getCoveringModuleModel('LIGHT', module_key));
+
+        this.currentSeatModel.update();
+        this.currentRadioModel.update();
+        this.currentAudioModel.update();
+        this.currentHMISettingsModel.update();
+    },
+
+    /**
+     * @function action
+     * @description Function used to do specified action in specified model
+     * @param {Event} event
+     */
+    action: function(event) {
+        this[event.model][event.method](event);
+    },
+
+    /**
+     * @function setInteriorVehicleData
+     * @param {Object} data
+     * @description Set data from mobile request by moduleId
+     */
+    setInteriorVehicleData: function(data) {
+        var moduleUUId = data.params.moduleData.moduleId;
+        var moduleType = data.params.moduleData.moduleType;
+
+        var moduleId = null;
```

Comment:
The second identifier used for the `moduleUUIDMapping` is a location grid coordinate as a string `L<level>R<row>C<col>` so `moduleId` should be renamed to 
```suggestion
        var location_name = null;
```

Reply:
@ShobhitAd accepted. Fixed in https://github.com/smartdevicelink/sdl_hmi/commit/3356c1d8aa1ec7b881f8a7e25bb4cde2a50033af